Formula/openmsx.rb

```diff
@@ -20,31 +19,42 @@ class Openmsx < Formula
     sha256 cellar: :any, high_sierra: "86fff1a90fff96cb0398184ef7ebdeb804edda4c9d34ee5e7278159df64b10e3"
   end
 
+  depends_on "python@3.9" => :build
   depends_on "freetype"
   depends_on "glew"
+  depends_on "libogg"
   depends_on "libpng"
-  depends_on "sdl"
-  depends_on "sdl_ttf"
+  depends_on "libvorbis"
+  depends_on "sdl2"
+  depends_on "sdl2_ttf"
+  depends_on "theora"
 
-  def install
-    # Fixes a clang crash; this is an LLVM/Apple bug, not an openmsx bug
-    # https://github.com/Homebrew/homebrew-core/pull/9753
-    # Filed with Apple: rdar://30475877
-    ENV.O0
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "alsa-lib"
+  end
 
+  def install
     # Hardcode prefix
     inreplace "build/custom.mk", "/opt/openMSX", prefix
+    inreplace "build/probe.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
Not sure if there is an alternative for non-default prefix: https://github.com/openMSX/openMSX/blob/RELEASE_17_0/build/probe.py#L368

Another option I found still required `inreplace` but on `configure`: https://github.com/openMSX/openMSX/blob/master/configure#L32
In this case, the following might work:
```
$MAKE -f build/main.mk probe 3RDPARTY_INSTALL_DIR=#{HOMEBREW_PREFIX}
```